.changeset/moody-pandas-fall.md

```diff
@@ -1,11 +1,12 @@
 ---
 "@fluidframework/tree": minor
+"fluid-framework": minor
 ---
 ---
 "section": tree
 ---
 
-Schema evolution now supports adding optional fields to object node types without staged rollout using alpha APIs.
```

Comment:
The phrasing here was ambiguous: it could be read as we no longer need staged rollouts using alpha APIs (implying APIs for doing staged rollouts are now stable).